lib/logstash/runner.rb

```diff
@@ -63,15 +44,6 @@ def run(args)
         end
         return LogStash::Agent.run($0, agent_args)
       end,
-      "rspec" => lambda do
```

Comment:
@jsvd why not? just "because it works on 1.4.2" is not a good enough reason IMO. This is not a major UI change and will not affect our "normal" users. It affects "only" those that want to run specs, and seriously, the developers will understand that change.